samples/subsys/nvs/sample.yaml

```diff
@@ -21,3 +21,27 @@ tests:
            39 3a 3b 3c 3d 3e 3f 40 41 42 43 44 45 46 47 48 49 4a 4b 4c 4d 4e 4f 50 51 52 53 54 55 \
            56 57 58 59 5a 5b 5c 5d 5e 5f 60 61 62 63 64 65 66 67 68 69 6a 6b 6c 6d 6e 6f 70 71 72 \
            73 74 75 76 77 78 79 7a 7b 7c 7d 7e 7f"
+  sample.nvs.internal_flash:
+    tags: nvs
+    depends_on: nvs
```

Comment:
we need add nvs in the <board>.yaml support list, otherwise this application will be filtered out.

Reply:
Thanks @hakehuang ,
I already have `nvs` added to frdm_mcxn947_mcxn947_cpu0.yaml.  And I confirmed Twister will run this sample on these 2 boards.  Or are you referring to something else?

Reply:
Thanks. So only test on frdm board is enough.